client/src/components/WorkspacePanels/MainWorkspacePanel/MainWorkspace.js

```diff
@@ -1,5 +1,10 @@
+<<<<<<< HEAD
+import React, { useState } from "react";
+import { Droppable, Draggable } from "react-beautiful-dnd";
+=======
 import React from "react";
 import { Droppable } from "react-beautiful-dnd";
+>>>>>>> cd0d1ae406ded26a1428453e49b186ff980e154c
```

Comment:
I don't think these lines should be there, but I'll merge it and fix it rn.

Reply:
Oops thank you for catching it!